src/model/enums/enums.rs

```diff
@@ -0,0 +1,20 @@
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum ColorTheme {
+    Dark,
+    Light,
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum CultureId {
+    EnUS,
+}
```

Comment:
NIT: we could use pub struct CultureID(u8) and operate on u8 for better performance.